payloads/external/Makefile.inc

```diff
@@ -202,6 +202,8 @@ $(obj)/UEFIPAYLOAD.fd edk2: $(DOTCONFIG) $(IPXE_EFI)
 		CONFIG_EDK2_DASHARO_INTEL_ME_OPTIONS=$(CONFIG_EDK2_DASHARO_INTEL_ME_OPTIONS) \
 		CONFIG_EDK2_DASHARO_USB_CONFIG=$(CONFIG_EDK2_DASHARO_USB_CONFIG) \
 		CONFIG_EDK2_DASHARO_NETWORK_CONFIG=$(CONFIG_EDK2_DASHARO_NETWORK_CONFIG) \
+		CONFIG_EDK2_DASHARO_NETWORK_BOOT_DEFAULT_ENABLE=$(CONFIG_EDK2_DASHARO_NETWORK_BOOT_DEFAULT_ENABLE) \
```

Comment:
@3mdeb-karolzmyslowski About this commit message:

`Makefile.inc, Kconfig, Makefile: Disable network boot, added .config `

Since it simply adds network boot enable option into edk2, I would name it:

`payloads/external/edk2: introduce DASHARO_NETWORK_BOOT_DEFAULT_ENABLE`, or similar.

Reply:
OK, but how I could correct it? By option `--amend`? By squashing?